portality/settings.py

```diff
@@ -163,6 +163,7 @@
 # directory to upload files to.  MUST be full absolute path
 # The default takes the directory above this, and then down in to "upload"
 UPLOAD_DIR = os.path.join(ROOT_DIR, "upload")
+UPLOAD_ASYNC_DIR = os.path.join(ROOT_DIR, "upload_async")
```

Comment:
Why do we need two different directories for the uploads?

Reply:
`UPLOAD_DIR` for website xml upload and ingestarticles background job